src/Razor/test/Microsoft.CodeAnalysis.Remote.Razor.Test/TagHelperDeltaResultTest.cs

```diff
@@ -10,13 +10,8 @@
 
 namespace Microsoft.CodeAnalysis.Remote.Razor;
 
-public class TagHelperDeltaResultTest : TagHelperDescriptorTestBase
+public class TagHelperDeltaResultTest(ITestOutputHelper testOutput) : TagHelperDescriptorTestBase(testOutput)
```

Comment:
```suggestion
public class TagHelperDeltaResultTest(ITestOutputHelper testOutput) 
    : TagHelperDescriptorTestBase(testOutput)
```

Reply:
I don't know that a code style of always moving base types to the next line when a primary constructor is all that useful. In this case, the line is shortish and doesn't wrap, and removes significant boilerplate.

Reply:
I'm thinking only for primary constructors, so it follows similar to constructors

```csharp
public class MyClass(string s)
    : base(s)
{
}
```

it's all very new though, so it's just a thought and not a well formulated guideline

Reply:
> I'm thinking only for primary constructors, so it follows similar to constructors

I understand where you're coming from. There's definitely a correspondence with base constructor calls. However, there's also a correspondence with base types that don't have primary constructors, and a correspondence with existing positional records with a primary constructor that calls base. In fact, because of records, this syntax has existed since C# 9. :smile: The "correct" style isn't immediately obvious because there could also be interfaces after the base call -- do those move to yet another line? If so, why aren't we doing that for classes without primary constructors? 🤷 

I would say we should follow whatever the Roslyn repo is doing WRT code style on primary constructors, which might be flexible. However, for these test classes, adding a line break doesn't add any value and just expands pure boilerplate. So, I'd advocate for leaving them as is.

Reply:
> I would say we should follow whatever the Roslyn repo is doing WRT code style

Clearly its been too long since you've been in a meeting where Roslyn folks try to agree on code style 😛

Reply:
Ha! Sorry about that. I meant the Roslyn IDE portion -- the code that Cyrus is keeping modern. 🤣